docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,359 @@
+----------------------
+The ``regional_workflow`` repository must not contain source code for compiled programs. Only scripts, configuration files, and documentation should reside in this repository. 
+
+**General Coding Standards:** 
+    * All bash scripts must explicitly be ``#!/bin/bash`` scripts. They should not be login-enabled.
+    * MacOS does not have all Linux utilities by default. Developers should ensure that they do not break any MacOS capabilities with their contribution.
+    * All code must be indented appropriately and conform to the style of existing scripts (e.g., local variables should be lowercase, global variables should be uppercase).
+
+..
+    COMMENT: And what is this style? Do we have a style guide? Ask about linking to this `style guide <https://google.github.io/styleguide/shellguide.html>`__
+
+**Python Coding Standards:** 
+    * All newly added Python code must be linted with a score of 10/10 following the .pylintrc configuration file set by the code managers. 
+    * All Python code contributions should come with an appropriate ``environment.yml`` file for the feature. 
+    * Keep the use of external Python packages to a minimum for necessary workflow tasks. Currently these include ``f90nml``, ``pyyaml``, and ``jinja``. 
+
+**Workflow Design:** Follow the `NCO Guidelines <https://www.nco.ncep.noaa.gov/idsb/implementation_standards/>`__ for what is incorporated in each layer of the workflow. This is particularly important in the ``scripts`` directory. 
+
+**Module files (env files):** All official platforms should have an environment file that can be sourced to provide the appropriate python packages and other settings for the platform. 
+
+**Management of the Configuration File:** New configurable options must be consistent with existing configurable options. Add necessary checks on acceptable options where applicable. Add appropriate default values.
+
+**Management of Template Files:** If a new configurable option is required in an existing template, it must be handled similarly to its counterparts in the scripts that fill in the template.
+
+..
+    COMMENT: Add Jinja template info when available.
+
+**Namelist Management:** Namelists in ``ufs-srweather-app`` and ``regional_workflow`` are generated using a Python tool and managed by setting YAML configuration parameters. This allows for the management of multiple configuration settings with maximum flexibility and minimum duplication of information. 
+
+
+
+Contributor Requirements
+===========================
+
+Preparing Code for Contribution to the UFS SRW Application
+------------------------------------------------------------
+
+All changes should be associated with a GitHub Issue. If a developer is working on a change, they should search the existing issues in the appropriate repository (``ufs-srweather-app`` and/or ``regional_workflow``). If an issue does not exist for the work they are doing, they should create one prior to opening a new pull request.
```

Comment:
I think we can get around this problem by modifying the text to read, "Developers should X and Y".  Then the "they" would be appropriate, since "developers" is plural.